src/servers/graphql-main-server.ts

```diff
@@ -164,8 +169,24 @@ const resolvers = {
     getWalletFees: () => ({
       deposit: yamlConfig.fees.deposit,
     }),
+    registerCaptchaGeetest: async (_, __, { logger, ip }) =>
+      await registerCaptchaGeetest({ logger, ip }),
   },
   Mutation: {
+    requestPhoneCodeGeetest: async (
+      _,
+      { phone, geetestChallenge, geetestValidate, geetestSeccode },
+      { logger, ip },
+    ) => ({
+      success: await requestPhoneCodeGeetest({
+        phone,
+        geetestChallenge,
+        geetestValidate,
+        geetestSeccode,
+        logger,
```

Comment:
@jrbuhl93 Notice the rename of the parameters, now these start with geetest instead of captcha.